runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -0,0 +1,114 @@
+import asyncio
+import os
+import json
+import contextvars
+import functools
+from asyncio import AbstractEventLoop
+from importlib import import_module
+from typing import Optional, Callable, Awaitable, Dict
+from distutils.util import strtobool
+
+from pydantic import BaseSettings
+from mlserver.errors import MLServerError
+
+from optimum.onnxruntime import (
+    ORTModelForCausalLM,
+    ORTModelForFeatureExtraction,
+    ORTModelForQuestionAnswering,
+    ORTModelForSequenceClassification,
+    ORTModelForTokenClassification,
+)
+
+
+HUGGINGFACE_TASK_TAG = "task"
+
+ENV_PREFIX_HUGGINGFACE_SETTINGS = "MLSERVER_MODEL_HUGGINGFACE_"
+HUGGINGFACE_PARAMETERS_TAG = "huggingface_parameters"
+PARAMETERS_ENV_NAME = "PREDICTIVE_UNIT_PARAMETERS"
+
+SUPPORTED_OPTIMIZED_TASKS = {
+    "feature-extraction": ORTModelForFeatureExtraction,
+    "sentiment-analysis": ORTModelForSequenceClassification,
+    "ner": ORTModelForTokenClassification,
+    "question-answering": ORTModelForQuestionAnswering,
+    "text-generation": ORTModelForCausalLM,
+}
+
+
+class RemoteInferenceError(MLServerError):
+    def __init__(self, code: int, reason: str):
+        super().__init__(
+            f"Remote inference call failed with {code}, {reason}", status_code=code
+        )
+
+
+class InvalidTranformerInitialisation(MLServerError):
+    def __init__(self, code: int, reason: str):
+        super().__init__(
+            f"Remote inference call failed with {code}, {reason}",
+            status_code=code,
+        )
```

Comment:
Does the runtime use these errors?

Reply:
Actually, just saw that `InvalidTransformerInitialisation` is used, but looks like the error message was copied over from the other one.